app/controllers/application_controller.rb

```diff
@@ -5,6 +5,10 @@ class ApplicationController < ActionController::Base
   before_action :configure_permitted_parameters, if: :devise_controller?
   # check_authorization unless: :devise_controller?
 
+  if :devise_controller?
+    layout 'center_form'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.